docs/cookbook/images-on-entity.rst

```diff
@@ -286,6 +285,24 @@ set the ``allow_add`` option to ``true``.
         }
     }
 
+.. tip::
+
+    In case you need only a single image upload, this can be done in 2 very easy steps.
+    
+    First, in the code for the form provided above set ``allow_add`` to ``false``. You may want to set ``allow_delete`` to ``false`` too because if user deletes the image, there is no way to add a new one until the form is saved and page is reloaded.
```

Comment:
Changed the whole block, as I think if you need a single image, you set both to ``false``